### PR TITLE
Add premium flag option to catalog loading code

### PR DESCRIPTION
**Pull request by arbulu89**
Add `premium` field to the checks speficication

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]
Premium :tm:

**Review by EMaksy — COMMENTED** [review]
LGTM :D

File: test/support/factory.ex [hunk 1]
```diff
@@ -26,7 +26,8 @@ defmodule Wanda.Factory do
       severity: Enum.random([:critical, :warning, :passing]),
       facts: build_list(10, :catalog_fact),
       values: build_list(10, :catalog_value),
-      expectations: build_list(10, :catalog_expectation)
+      expectations: build_list(10, :catalog_expectation),
+      premium: false
```
Comment (dottorblaster) [inline test/support/factory.ex:30]:
```suggestion
      premium: Enum.random([true, false])
```

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a tiny change request then I think we can merge

File: test/support/factory.ex (hunk 1 above)
Comment (arbulu89) [reply test/support/factory.ex]: Done

**Review by fabriziosestito — APPROVED** [review]

**Pull request merged**